addons/resource/models/resource.py

```diff
@@ -944,16 +944,16 @@ def _adjust_to_calendar(self, start, end):
         end, revert_end_tz = make_aware(end)
```

Comment:
Start and end are in the user timezone without specified timezone.
For a user located in Belgium, it will be:
start: day - 1, 22:00
end: day, 21:59:59